src/pymor/models/interface.py

```diff
@@ -205,6 +271,12 @@ def compute(self, solution=False, output=False,
             If `True`, return the model's internal state.
         output
             If `True`, return the model output.
+        solution_d_mu
+            If not `False`, either `True` to return the derivative of the model's
+            internal state w.r.t. all parameter components or a tuple `(parameter, index)`
+            to return the derivative of a single parameter component.
+        output_d_mu
+            If `True`, return the gradient of the model output w.r.t. the |Parameter|.
```

Comment:
Wouldn't it make sense to also allow compute the sensitivity w.r.t. a specific direction here?

Reply:
I was planning to work on that in the future but I do not intend to add it to this PR, since the d_mu is only needed for the gradient so far. For the hessian, e.g. , a sensitivity to an arbitrary direction would be desired.

Reply:
Ok.